plugins/source/azuredevops/codegen/recipes/core.go

```diff
@@ -0,0 +1,20 @@
+package recipes
+
+import (
+	"github.com/microsoft/azure-devops-go-api/azuredevops/v6/core"
+)
+
+func Core() []*Resource {
+	resources := []*Resource{
+		{
+			SubService: "projects",
+			Struct:     core.TeamProjectReference{},
```

Comment:
curious, why not a pointer? VIa `new` or `&`?
(in other places we use pointers)

Reply:
Why aren't you using the proper type [TeamProject](https://pkg.go.dev/github.com/microsoft/azure-devops-go-api/azuredevops/v6/core#TeamProject) instead of reference (stripped down)?

I presume, list ill get the references, but then you'll need to gen preresolver and replace the item.

Basically, reference doesn't have properties and collections.

[GetProject](https://pkg.go.dev/github.com/microsoft/azure-devops-go-api/azuredevops/v6/core#ClientImpl.GetProject)
this, on the other hands, returns a [TeamProject](https://pkg.go.dev/github.com/microsoft/azure-devops-go-api/azuredevops/v6/core#TeamProject).
You'll still have to add an extra column for [GetProjectProperties](https://pkg.go.dev/github.com/microsoft/azure-devops-go-api/azuredevops/v6/core#ClientImpl.GetProjectProperties).
And a relation for collentions.

Reply:
> curious, why not a pointer?

Good point, I'll change it to a pointer for consistency.

> I presume, list ill get the references

Correct, the list operation returns the references in 1 API call.
Not sure we want to add the additional information unless someone requests it, as it requires additional API calls

Reply:
At the very least [TeamProject](https://pkg.go.dev/github.com/microsoft/azure-devops-go-api/azuredevops/v6/core#TeamProject) has `Capabilities` field (also, `Links` field, but maybe we would want to skip it for now) that might be of interest, as well as `DefaultTeam`.

Reply:
decide to postpone till resources impl